### ISSUE TITLE
Add  GP account info to the WP users screen

### ISSUE THREAD

**Issue by Zodiac1978** [GlotPress/GlotPress-WP#918]
For example we could add a true/false info if this user has GlotPress admin privileges.

And we could add all locales/projects for which this user has validator rights. Like seen on https://de.wordpress.org in the entry "Translation Editors".

Maybe this is better optional to not clutter this table or we can use a custom field to store this information. Then we could use something like Admin Columns to easily add this column ourself.

**Comment by toolstack** [comment]
All of that information is already contained in the gp_permissions table.

Modifying the base WP tables would not be a good idea.

**Comment by ocean90** [comment]
I think this is about the list tables in wp-admin and not database tables?

**Comment by Zodiac1978** [comment]
Yes. I'm obviously failing in writing understandable issues here. :/

**Comment by toolstack** [comment]
Ah, in that case yes, it would be doable.
